addons/website/static/src/snippets/s_share/000.js

```diff
@@ -38,8 +38,10 @@ const ShareWidget = publicWidget.Widget.extend({
             });
             if ($a.attr('target') && $a.attr('target').match(/_blank/i) && !$a.closest('.o_editable').length) {
                 $a.on('click', function () {
-                    window.open(this.href, '', 'menubar=no,toolbar=no,resizable=yes,scrollbars=yes,height=550,width=600');
-                    return false;
+                    if (!$a.closest('.editor_enable').length) {
```

Comment:
@qsm-odoo Is there a better way to retrieve the editable from the public widget?

Reply:
I've adapted the code to use the destroy 👍

Reply:
For history purposes, I checked what I explained to be sure:
- The `.off('click')` was indeed removed with 27166ff9dfa1b146c751b0aec388b9a8eb97d545 (so no dead code is left) :+1: 
- It was introduced with the original summernote editor addition with https://github.com/odoo/odoo/commit/9069d0127c176317436b67b23ae5677dd9d53de7 (so we'll never know the exact reason and like I said, I think it's not strictly needed and can be left as it is today) :+1: